registration-processor/stage-groups/stage-group-1/Dockerfile

```diff
@@ -33,23 +37,28 @@ ENV is_glowroot_env=${is_glowroot}
 # environment variable to pass artifactory url, at docker runtime
 ENV artifactory_url_env=${artifactory_url}
 
+ENV stage_group_jar_name_env=${stage_group_jar_name}
+
+ENV stage_group_name_env=${stage_group_name}
+
 # change volume to whichever storage directory you want to use for this container.
 VOLUME /home/logs
 
-COPY ./target/registration-processor-external-stage-*.jar registration-processor-external-stage.jar
+COPY ./target/${stage_group_jar_name_env}-*.jar ${stage_group_jar_name_env}.jar
 
 CMD if [ "$is_glowroot_env" = "present" ]; then \
     wget "${artifactory_url_env}"/artifactory/libs-release-local/io/mosip/testing/glowroot.zip ; \
     apt-get update && apt-get install -y unzip ; \
     unzip glowroot.zip ; \
     rm -rf glowroot.zip ; \
-    
     sed -i 's/<service_name>/registration-processor-external-stage/g' glowroot/glowroot.properties ; \
-    java -Dspring.cloud.config.label="${spring_config_label_env}" -Dspring.profiles.active="${active_profile_env}" -Dspring.cloud.config.uri="${spring_config_url_env}" -jar -javaagent:glowroot/glowroot.jar registration-processor-external-stage.jar ; \
+    java -Dspring.cloud.config.label="${spring_config_label_env}" -Dspring.profiles.active="${active_profile_env}" -Dspring.cloud.config.uri="${spring_config_url_env}" -Dstage-group-name=${stage_group_name_env} -jar -javaagent:glowroot/glowroot.jar ${stage_group_jar_name_env}.jar ; \
     else \
-    java -Dspring.cloud.config.label="${spring_config_label_env}" -Dspring.profiles.active="${active_profile_env}" -Dspring.cloud.config.uri="${spring_config_url_env}" -jar registration-processor-external-stage.jar ; \
+    java -Dspring.cloud.config.label="${spring_config_label_env}" -Dspring.profiles.active="${active_profile_env}" -Dspring.cloud.config.uri="${spring_config_url_env}" -Dstage-group-name=${stage_group_name_env} -jar ${stage_group_jar_name_env}.jar ; \
     fi
 
+# Sample Command to build:
```

Comment:
Can we put below build and run commands in the readme file?

Reply:
Ok.. Will add the readme and push.

Reply:
Added Readme.